test/nginx-static.conf

```diff
@@ -19,6 +19,11 @@ server {
         expires 1d;
     }
 
+    # no content for the favicon (the default 404 response is treated as HTML)
+    location /favicon.ico {
+        return 204;
+    }
```

Comment:
This will skip including the `/favicon.ico` response in the `gzipRequests ` and `htmlCount` metrics.